docs/runtimes/mlrun_code_annotations.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>If you use the <code># mlrun: start-code</code> and <code># mlrun: end-code</code> annotations multiple times in a notebook, the whole notebook will be scanned. &nbsp;The annotations can be named like the following example, and they can be nameless (if you choose nameless remember all nameless annotations in the notebook will be used).</p><p> </p><p>(<strong><em style="">Didn't you already state this above with the example of how to name the annotation? What is different here?)</em></strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>It was asked to emphasis that multi section function can work with nameless annotations as well (since the example uses named annotations).</p>